activation/activation.go

```
@@ -222,6 +232,44 @@ func (b *Builder) buildNipstChallenge(epoch types.EpochId) error {
 	return nil
 }
 
+func (b *Builder) StartPost(rewardAddress address.Address, logicalDrive string, commitmentSize uint64) error {
+	b.SetCoinbaseAccount(rewardAddress)
+
+	if !b.nipstBuilder.IsPostInitialized() {
```

If post is already initialized, it should return an error. There's no reason for the endpoint to continue swallowing these requests without reporting anything back.